src/store/modules/staticData.js

```diff
@@ -14,20 +17,32 @@ import * as getters from './getters/staticData';
 import * as actions from './actions/staticData';
 
 const state = () => ({
-  userInfos: {},
+  userInfosById: {},
```

Comment:
Should remain unchanged for this PR?
```suggestion
  userInfos: {},
```

Reply:
right, thanks